src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -5964,9 +5995,10 @@ export class StudyViewPageStore
         }
 
         if (!pending && this.isInitiallLoad) {
-            pending = pending || this.selectedSamples.result.length === 0;
-        }
-        if (!pending && this.isInitiallLoad) {
+            pending =
+                pending ||
+                this.clinicalEventTypeCounts.isPending ||
```

Comment:
@alisman This is the one change I had to add since you reviewed. Needed to add this to prevent "out of sync" bug